vcpkg/overlay/qgis/vcpkg-cmake-wrapper.cmake

```
@@ -131,6 +131,12 @@ if(QGIS_FOUND AND "@VCPKG_LIBRARY_LINKAGE@" STREQUAL "static")
   _find_and_link_library(spatialindex qgis_core spatialindex spatialindex-64)
   _find_and_link_library(freexl qgis_core)
   _find_and_link_library(qt5keychain qgis_core)
+  _find_and_link_library(poppler qgis_core)
+  _find_and_link_library(freetype qgis_core)
+  _find_and_link_library(freetype qgis_core)
+  _find_and_link_library(libbrotlicommon qgis_core)
+  _find_and_link_library(libbrotlidec qgis_core)
+  _find_and_link_library(libbrotlienc qgis_core)
```

```suggestion
  _find_and_link_library(brotlidec qgis_core)
  _find_and_link_library(brotlicommon qgis_core)
```

May be better like that (80% sure)